pymc3/tuning/starting.py

```diff
@@ -30,21 +34,21 @@ def find_MAP(start=None, vars=None, fmin=None,
     start : `dict` of parameter values (Defaults to `model.test_point`)
     vars : list
         List of variables to set to MAP point (Defaults to all continuous).
-    fmin : function
-        Optimization algorithm (Defaults to `scipy.optimize.fmin_bfgs` unless
```

Comment:
actually maybe its a bit trickier than I thought.  different fmin functions have slightly different APIs.  According to its docstring, `fmin_bfgs` doesnt support a cost function that returns both the value and gradient, while `fmin_l_bfgs_b` does.